### PR TITLE
bsd: lock root password before publish

### PR DESCRIPTION
**Pull request by nbyavuz**
Root's password needs to be cleared before publish.

### CODE REVIEW

File: packer/bsd/netbsd_vanilla.pkr.hcl [hunk 1]
```diff
@@ -106,6 +106,14 @@ build {
     inline = ["chmod 744 /etc/rc.local && chmod 744 /etc/rc.shutdown"]
   }
 
+  provisioner "shell" {
+    inline = ["/usr/bin/sed -i 's/PermitRootLogin yes/PermitRootLogin prohibit-password/g' /etc/ssh/sshd_config"]
+  }
+
+  provisioner "shell" {
```
Comment (anarazel) [inline packer/bsd/netbsd_vanilla.pkr.hcl:113]: Please add a comment explaining what this does, -C isn't that explanatory.

File: packer/bsd/openbsd_vanilla.pkr.hcl [hunk 2]
```diff
@@ -66,6 +66,14 @@ build {
     inline = ["chmod 744 /etc/rc.local && chmod 744 /etc/rc.shutdown"]
   }
 
+  provisioner "shell" {
+    inline = ["sed -i 's/PermitRootLogin yes/PermitRootLogin prohibit-password/g' /etc/ssh/sshd_config"]
+  }
+
+  provisioner "shell" {
```
Comment (anarazel) [inline packer/bsd/openbsd_vanilla.pkr.hcl:73]: See above re comment.

File: packer/bsd/netbsd_vanilla.pkr.hcl (hunk 1 above)
Comment (nbyavuz) [reply packer/bsd/netbsd_vanilla.pkr.hcl:139]: Done.

File: packer/bsd/openbsd_vanilla.pkr.hcl (hunk 2 above)
Comment (nbyavuz) [reply packer/bsd/openbsd_vanilla.pkr.hcl:99]: Done.

**Review by anarazel — APPROVED** [review]

**Pull request merged**